Blueprint: Themes icon change and CSS updates for the uploaded filename

moon0326 opened the pull request:
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Changes proposed in this Pull Request:

<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->

<!-- Describe the changes made to this Pull Request and the reason for such changes. -->

<!-- For bug fixes: If known, please provide links to help with traceability and escape analysis. -->
<!-- Please include a link to the issue of the bug being fixed, if one doesn't exist please create it. -->
<!-- If the PR that introduced the bug is known, please also add its link below. -->

Closes #56405 #56406

This PR updates:

1. Replaced `Themes` icon with `layout` from @wordpress/icons.
2. Updated styles for uploaded files.



### Screenshots or screen recordings:

<!-- If this PR includes UI changes, please provide screenshots or a screen recording for clarity. -->
<!-- This section can be removed if not applicable. -->

#56405

| Before | After |
| ------ | ----- |
|   ![Screenshot 2025-03-14 at 11 42 44 AM](https://github.com/user-attachments/assets/d16868dd-3f90-4119-9c3a-0d1c35ef8e4e) |   ![Screenshot 2025-03-14 at 11 39 31 AM](https://github.com/user-attachments/assets/cf4b1a74-0206-428f-a142-cdb3a850f37c)    |

#56406

| Before | After |
| ------ | ----- |
|  ![Screenshot 2025-03-14 at 11 43 11 AM](https://github.com/user-attachments/assets/ddb22af1-3ce6-4f5a-aa41-dbc9a7a52ddf)  |   ![Screenshot 2025-03-14 at 11 39 42 AM](https://github.com/user-attachments/assets/42a328ad-8c6d-474b-b455-ae0b2fa3d202)
    |


<!-- Begin testing instructions -->

### How to test the changes in this Pull Request:

<!-- Include detailed instructions on how these changes can be tested. Review and follow the guide for how to write high-quality testing instructions. -->

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Navigate to WooCommerce -> Settings -> Advanced -> Features and enable Blueprint (Beta)
3. Navigate to WooCommerce -> Settings -> Advanced -> Blueprint (beta)
4. Confirm the icon change.
5. Click "Export" to generate a schema file.
6. Drag and drop the imported schema file.
7. Confirm the uploaded file box style changes.

<!-- End testing instructions -->

### Testing that has already taken place:

<!-- Detail any testing that has already been conducted. -->
<!-- Include environment details such as hosting type, plugins, theme, store size, store age, and relevant settings. -->
<!-- Mention any analysis performed, such as assessing potential impacts on environment attributes and other plugins, performance profiling, or LLM/AI-based analysis. -->
<!-- Within the testing details you provide, please ensure that no sensitive information (such as API keys, passwords, user data, etc.) is included in this public pull request. -->

### Changelog entry

<!-- You can optionally choose to enter a changelog entry by checking the box below and supplying data. -->
<!-- It will trigger the 'Add changelog to PR' CI job to create and push the entry into the branch. -->

<!-- Due to org permissions, the job might fail for PRs crated from a fork under GitHub organizations. Possible solutions: -->
<!-- * Create entry manually with `pnpm --filter='@woocommerce/plugin-woocommerce' changelog add` and push it into the branch (replace `@woocommerce/plugin-woocommerce` with package name from nearest `package.json` file) -->
<!-- * Create entry from supplied PR data and push it automatically `pnpm utils changefile pr-number-here -o github-org-name-here` -->

-   [x] Automatically create a changelog entry from the details below.

<!-- If no changelog entry is required for this PR, you can specify that below and provide a comment explaining why. This cannot be used if you selected the option to automatically create a changelog entry above. -->

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

<!-- Choose only one -->

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

<!-- Choose only one -->

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message <!-- Add a changelog message here -->

Blueprint: replace 'Themes' icon and updated styles for the uploaded filename box.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment <!-- If your Pull Request doesn't require a changelog entry, a comment explaining why is required instead -->

</details>

Commit:
Change `themes` icon to layout

Commit:
Fix import success styles - match the default dropzone form

ilyasfoo reviewed (commented):
It seems there's more styling discrepancy with Figma (Y5pUYSJPsGEud1vknUZhi8-fi-4023_24151) when error message is shown. I think margin, font, & color should be fixed, but dismiss button could be a follow-up since it requires functionality change.

| Figma    | PR |
| -------- | ------- |
| <img width="659" alt="image" src="https://github.com/user-attachments/assets/83f220ac-3646-46d6-8d92-bc741ee4bf07" /> | <img width="659" alt="image" src="https://github.com/user-attachments/assets/04805363-a09a-4777-9a6f-81b97f58306c" /> |

Commit:
Fix broken test

Commit:
Style updates

* Fix error notice styles and add dismiss button
* Use #2f2f2f as a base text color

moon0326 commented:
> It seems there's more styling discrepancy with Figma ([Y5pUYSJPsGEud1vknUZhi8-fi-4023_24151](https://href.li/?https://www.figma.com/file/Y5pUYSJPsGEud1vknUZhi8/?node-id=4023-24151)) when error message is shown. I think margin, font, & color should be fixed, but dismiss button could be a follow-up since it requires functionality change.
> 
> Figma	PR
> <img alt="image" width="659" src="https://private-user-images.githubusercontent.com/3747241/423311392-83f220ac-3646-46d6-8d92-bc741ee4bf07.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIyMzk2MDgsIm5iZiI6MTc0MjIzOTMwOCwicGF0aCI6Ii8zNzQ3MjQxLzQyMzMxMTM5Mi04M2YyMjBhYy0zNjQ2LTQ2ZDYtOGQ5Mi1iYzc0MWVlNGJmMDcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTdUMTkyMTQ4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGYzZjAwZGFjYzc3YWVlZmJkM2FmOTI1M2NhMDBjOGZiZThhZTdhNjFmM2E1MzZlNGM0NWM0ZTg5YmU3MjgyYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Q9gcOw7hcnKy-QF6bIbI90MSE2cX_Wucufy2c4cnK3I">	<img alt="image" width="659" src="https://private-user-images.githubusercontent.com/3747241/423312447-04805363-a09a-4777-9a6f-81b97f58306c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIyMzk2MDgsIm5iZiI6MTc0MjIzOTMwOCwicGF0aCI6Ii8zNzQ3MjQxLzQyMzMxMjQ0Ny0wNDgwNTM2My1hMDlhLTQ3NzctOWE2Zi04MWI5N2Y1ODMwNmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTdUMTkyMTQ4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzFjNTIxYjAyOTk3NGMyZmJlNjIxOTk4YzNjNGZjY2Q5ZWM2NjEzYzIzY2U2NDZkZjQwZWEwY2FjOTg5YjRlYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.qKbHr-WW7QX3Cs9eBLg-BDyAHF85FSwYjZhwrkC23fk">

Good catch!

Fixed it in [Style updates](https://github.com/woocommerce/woocommerce/pull/56411/commits/fb6174e34db86148efc6cc2e89505a9b8b3477a1)

I've added the dismiss button as it is a part of `Notice` component. 
![Screenshot 2025-03-17 at 12 48 00 PM](https://github.com/user-attachments/assets/08b5c7db-e627-4160-9ce3-a12360eb52ee)

Commit:
Merge branch 'trunk' into update/blueprint-style-updates

Commit:
Merge branch 'trunk' into update/blueprint-style-updates

Commit:
Add changefile(s) from automation for the following project(s): woocommerce, woocommerce/client/admin

ilyasfoo reviewed (approved):
Nice, thanks for addressing, this is looking great! I found another minor thing, it seems while uploading a file, the import component disappears for a very short while. I think this is optional to address since it has minimal UX impact:

https://github.com/user-attachments/assets/0f47e5b0-1b6e-4e51-bfd9-d40fdfff5798

#### Review checklist

| Parameter  | Result |
| ------------- | ------------- |
| Approach  | ✅ All good |
| Implementation  | ✅ All good |
| Changelog  | ✅ All good |
| Documentation  | Not applicable |
| PR description  | ✅ All good |
| Addresses issue criteria  | ✅ Yes |

#### Scenarios tested

| Scenario  | Result | Screenshot |
| ------------- | ------------- | ------------- |
| Import blueprint UI | ✅ UI looks good | <img width="649" alt="image" src="https://github.com/user-attachments/assets/8f3c3f41-89e9-44c1-9b12-0d6f1828f18f" /> |
| Updated themes icon  | ✅ Icon looks correct | <img width="668" alt="image" src="https://github.com/user-attachments/assets/f6ee0477-0bf3-4c09-a7a3-bca55bc03275" /> |
| Error notice | ✅ UI looks good and functions correctly | <img width="656" alt="image" src="https://github.com/user-attachments/assets/f4fd573f-e654-4f7d-bf10-fdf6753ae9f2" /> |

chihsuan reviewed (approved):
LGTM! 👍 I verified UI changes look good.

This isn't related to the PR, but I observed that the file input cannot be cleared after importing the blueprint when the `x` button is clicked.

https://github.com/user-attachments/assets/216e1533-e69a-4ba3-b80c-9ddd96bd0cfc

#### Scenarios tested 

| Scenario | Result | Notes |
| -------- | ------ | ----- |
| Import blueprint UI | ✅ UI looks good | Verified with screenshots |
| Updated themes icon | ✅ Icon looks correct | Using standard WordPress icon |
| Error notice | ✅ UI looks good | Functions correctly |

moon0326 commented:
> LGTM! 👍 I verified UI changes look good.
> 
> This isn't related to the PR, but I observed that the file input cannot be cleared after importing the blueprint when the `x` button is clicked.


Thank you!

That's actually fixed in https://github.com/woocommerce/woocommerce/pull/56408

Commit:
Render the upload form in `parsingSteps` state

moon0326 commented:
> Nice, thanks for addressing, this is looking great! I found another minor thing, it seems while uploading a file, the import component disappears for a very short while. I think this is optional to address since it has minimal UX impact:

Good catch! Fixed it in [Render the upload form in parsingSteps state](https://github.com/woocommerce/woocommerce/pull/56411/commits/02edf4fa14cbea5db893a0952c4fe439911723bf)

Commit:
Merge branch 'trunk' into update/blueprint-style-updates

Commit:
Merge branch 'trunk' into update/blueprint-style-updates

Commit:
Merge branch 'trunk' into update/blueprint-style-updates

Commit:
Merge branch 'trunk' into update/blueprint-style-updates

Commit:
Merge branch 'trunk' into update/blueprint-style-updates
